src/q1/algebre-LEPL1101/summary/algebre-LEPL1101-summary.tex

```diff
@@ -0,0 +1,1023 @@
+% !TEX encoding = UTF-8 Unicode
+\documentclass[a4paper]{book}
+\usepackage[utf8]{inputenc}
+\usepackage[T1]{fontenc} 
+\usepackage[francais]{babel}
+\usepackage{layout}
```

Comment:
Ah oui, mais en fait tu n'as pas utilsé la fonction hypertitle comme dans le template, reagrde dans la synthèse que tu as créée vide: \hypertitle{analyse}{1}{LEPL}{1102}{Author1\and Author2\and Author3}{Professor} :smile: